### DISCUSSION TITLE
Self signed certificate connection error

### DISCUSSION THREAD

**Discussion by huck2007** [lightdash/lightdash#1728] [Q&A]
Hi,

I've just downloaded Lightdash to give it a test drive, and after successfully installing (on my macbook pro) I've hit a snag when setting up connections to git & Postgres.

I've got a private git repo with a dbt project deployed to it (produced using dbt cloud), so I've followed the instructions in the Lightdash docs to create a Personal Access Token (PAT) and provided those details along with the repo location details.     I've then added my connection details for a Postgres data warehouse (RDS instance in AWS).

When I click the button to test & save the connection I get the following error:

"There was an error getting connected
self signed certificate in certificate chain"

I'm not sure whether this is pointing towards a problem with my git setup or the database connection setup.    Any ideas?

Thanks
Scott

**Comment by owlas** [comment]
Hey @huck2007 ! 

I think this is an error with Lightdash not providing the correct AWS certificates needed to connect to postgres (RDS) on AWS. Let me look into this, we already have a solution for redshift, I imagine this is similar.

Are you still having this problem?

**Reply by owlas** [reply]
Tracking this as an issue here: https://github.com/lightdash/lightdash/issues/1816

**Reply by ZeRego** [reply]
@huck2007 What if you use the redshift form instead of the postgres form ? the only difference in our code between those two is that we add the amazon certificate 🤔  so it might work
